quickstart/101-virtual-network-create-two-subnets/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "resource_group_location" {
+  type        = string
+  default     = "eastus"
+  description = "Location of the resource group."
+}
+
+variable "resource_group_name_prefix" {
+  type        = string
+  default     = "rg"
+  description = "Prefix of the resource group name that's combined with a random ID so name is unique in your Azure subscription."
+}
+
+variable "vnet_address_space" {
```

Comment:
It seems like an unused variable, right?

Reply:
Yep, you are right.  I removed and sent up the changes.